tests/test-cases/Group1-Docker-Commands/1-21-Docker-Volume-LS.md

```diff
@@ -12,23 +12,24 @@ This test requires that a vSphere server is running and available
 
 #Test Steps:
 1. Deploy VIC appliance to vSphere server
-2. Issue docker create --name=test to the VIC appliance
-3. Issue docker volume ls to the VIC appliance
-4. Issue docker volume ls -q to the VIC appliance
-5. Issue docker volume ls -f dangling=true to the VIC appliance
-6. Issue docker volume ls -f name=test to the VIC appliance
-7. Issue docker create -v test:/test busybox to the VIC appliance
-8. Issue docker volume ls -f dangling=true to the VIC appliance
+2. Issue docker volume create --name=test to the VIC appliance
+3. Issue docker volume create --name=test2 to the VIC appliance
+4. Issue docker create -v test:/test busybox to the VIC appliance
+5. Issue docker volume ls to the VIC appliance
+6. Issue docker volume ls -q to the VIC appliance
+7. Issue docker volume ls -f dangling=true to the VIC appliance
```

Comment:
I will need to check these a little more. I have not added code for `-f` is not actually implemented yet so the filter will not be applied. Output will still come out as if a `docker volume ls` were issued. `image ls` works the same way at the moment.